catalog.html

```diff
@@ -0,0 +1,247 @@
+<!DOCTYPE html>
+<html lang="ru">
+  <head>
+    <meta charset="utf-8">
+    <title>Гостинницы</title>
+  </head>
+  <body>
+    <header class="page-header">
+        <nav class="navigation">
+          <ul class="navigation-list">
+           <a class="navigation-logo" href="index.html">
+            <img class="logo" src="img/logo.svg" alt="logo Седона">
+           </a>
```

Comment:
```suggestion
           <a class="navigation-logo" href="index.html">
              <img class="logo" src="img/logo.svg" alt="logo Седона">
           </a>
```
Я бы вынес лого из списка, во всяком случае пока что. Если потом решишь, что в списке его удобнее делать, то можешь обратно внести, но кажется, что ссылки все одинаковые, а лого немного отдельно